### DISCUSSION TITLE
RaspberryMatic not starting under Ubuntu 24.04 on Raspi 4B

### DISCUSSION THREAD

**Discussion by ChriZ982** [OpenCCU/OpenCCU#2771] [🇺🇸 Q&A]
### Describe the issue you are experiencing

We installed RaspberryMatic using the official Docker install script and started it. However, we are getting errors from `multimacd` and `ReGaHss` on startup. Also, the webpage only shows the forwarding message but nothing happens.

We kindly ask you to help us debug this issue as we have no idea what else to try and would love to use RaspberryMatic.

### Describe the behavior you expected

RaspberryMatic starting without errors and the webpage forwarding to the setup/login page.

### Steps to reproduce the issue

1. Install RaspberryMatic via the automatic Docker install script on Ubuntu 24.04
2. Start the ccu
3. Visit the webpage - it only shows that you will be forwarded but nothing happens

### What is the version this bug report is based on?

3.75.7-20240601

### Which base platform are you running?

rpi4 (RaspberryPi4, ARM64/aarch64)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
Identifying host system: Raspberry Pi 4 Model B Rev 1.4 (oci), OK
Initializing RTC Clock: no hardware found
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Running seedrng: OK
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Populating /dev using udev: done
Init onboard LEDs: init, OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.22.0.8, OK
Identifying Homematic RF-Hardware: ....HmRF: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:01:00.0-1.1.2, HmIP: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:01:00.0-1.1.2, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: 4.4.16=>4.4.18, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: ..........ERROR
Starting rfd: ERROR: /dev/mmd_bidcos missing, no BidCos-RF hardware found
Starting HMIPServer: ERROR: /dev/mmd_hmip missing
Starting ReGaHss: ....................ERROR
Starting CloudMatic: OK
Starting NeoServer: disabled
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.75.7.20240601 (raspmatic_oci_arm64)
```

### Additional information

uname -a
`Linux XYZXYZ 6.8.0-1005-raspi #5-Ubuntu SMP PREEMPT_DYNAMIC Wed May 22 15:01:35 UTC 2024 aarch64 aarch64 aarch64 GNU/Linux`

**Comment by jens-maus** [comment]
> We kindly ask you to help us debug this issue as we have no idea what else to try and would love to use RaspberryMatic.

This is not a help fora help, but a bug/issue ticket system where issues/bugs are reported that exist in RaspberryMatic. However, your issue is (as many others before you) certainly related to kernel module installation issues in your local system. So please read here (https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Docker-OCI#potential-container-startup-errors) and also in the discussion part and perhaps also ask for help in the homematic-forum.de for help.

**Comment by schacka** [comment]
I am also experiencing this error. Everything reinstalled. Modules load in the operating system without problems. In Docker /dev/mmd_* missing. The "problematic" setup uses a RPI-RF-MOD directly connected to the GPIOs of a rpi. Another setup uses a HB-RF-ETH --> no problems with 24.04.

There are probably changes in Ubuntu 24.04 that prevent the modules from loading in Docker using direct hardware access, maybe changed permissions (cgroup?). Unfortunately I am not skilled enough to help myself. Since the problem only occurs from Ubuntu 24.04 onwards, my solution is to stay on 23.10.
